front/public/humans.txt

```diff
@@ -0,0 +1,4 @@
+/* SITE */
+	Language: English
+	Doctype: HTML5
+    Technologies: HTML5, CSS3, ES2015, ember, ember-cli, pure, d3.js, babel, highcharts, loader.js, broccoli
```

Comment:
My understanding is that humans.txt is for describing the people involved in the project, but maybe that's not entirely accurate. Here's https://www.google.com/humans.txt for example:

> Google is built by a large team of engineers, designers, researchers, robots, and others in many different sites across the globe. It is updated continuously, and built with more tools and technologies than we can shake a stick at. If you'd like to help us out, see google.com/careers.

I also suspect that a manually-compiled list of technologies will be immediately subject to bitrot. Since this is an open source project with a public repository, I would think that people curious about the tech used could look at the package.json or Cargo.toml.

Is it common to use humans.txt in this way? Are there other examples of using it in the format you've used here?